src/pages/author/[...slug].tsx

```diff
@@ -82,23 +82,23 @@ const AuthorPage: NextPage<AuthorPageProps> = ({
     const titleAndLocation = getTitleAndLocation(title, location);
     const hasSocial = facebook || twitter || linkedin || youtube;
 
-    const LENGTH = articles.length;
-    const DATA = articles;
-    const LIMIT = 8;
-    const SHOW_MORE_INITIAL_STATE = LENGTH > LIMIT;
-
-    const [showMore, setShowMore] = useState(SHOW_MORE_INITIAL_STATE);
-    const [list, setList] = useState(DATA.slice(0, LIMIT));
-    const [index, setIndex] = useState(LIMIT);
-
-    const loadMore = () => {
-        const newIndex = index + LIMIT;
-        const newShowMore = newIndex <= LENGTH - 1;
-        const newList = list.concat(DATA.slice(index, newIndex));
-        setIndex(newIndex);
-        setList(newList);
-        setShowMore(newShowMore);
-    };
+    // const LENGTH = articles.length;
```

Comment:
I assume this is all temporary so that's why we are just commenting it out for now?

Reply:
yes it is. they want to avoid load more for seo purposes.